### PR TITLE
Open-Up/openup2018_01-barcamp#3 exclude load en.js from locale scripts

### PR DESCRIPTION
**Pull request by ThanhxD**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -185,7 +185,7 @@
     "time-grunt": "1.4.0"
   },
   "scripts": {
-    "postinstall": "./node_modules/.bin/bower install --allow-root",
+    "postinstall": "./node_modules/.bin/bower install --allow-root --production",
```
Comment (heroandtn3) [inline package.json]: this is out of scope of this PR, please revert it

Comment (tuanlc) [reply package.json]: He separated them into two commits.

Comment (heroandtn3) [reply package.json]: it's still out of scope of this PR

Comment (tuanlc) [reply package.json]: ok. So @ThanhxD please separate them into two pull request.

**Comment by tuanlc** [comment]
Hi,
I close this PR because there is no feedback from @ThanhxD And the issue is fixed.
